### ISSUE TITLE
Error image class

### ISSUE THREAD

**Issue by JustasKuizinas** [tjoskar/ng-lazyload-image#113]
Would be nice to add a class indicating that there was an error loading image.

**Comment by tjoskar** [comment]
I agree. Do you want to send me a pull request?

I think you can solve it by doing something like this:

```js
.catch(() => {
  if (errorImgPath) {
      setImage(image, errorImgPath);
  }
  image.className += ' ng2-failed-lazyloaded';
  return Observable.of(1);
})
```
at: https://github.com/tjoskar/ng2-lazyload-image/blob/9d4f6b7fc532e4995f188a08376a12bf1c2f460c/src/lazyload-image.ts#L72

**Comment by diogoqueiros24** [comment]
Hi @tjoskar . Thanks for the great job in this repo.

Your suggestion for the error class solved my needs and I took the liberty of doing a pull request #139 .

**Comment by tjoskar** [comment]
Closed with #139
